CHANGELOG.md

```diff
@@ -14,15 +14,15 @@
 
 ## Compatibility Notes
 
-- The methods `refreshTokenGrant`,`userTokenGrant` and `clientCredentialsGrant` of the `xsuaa-service` accept now a `string` or `XsuaaServiceCredentials` as first argument.
-This argument represents the `URL` used to fetch the access token from and represent two cases.
-A `string` values denotes the URL of the token service and it is not changed.
-A  `XsuaaServiceCredentials` denotes that the XSUAA is used to receive the token and a suffix `oauth/token` is appended to the URL property within `XsuaaServiceCredentials`.
-In the past the suffix was also appended for `string` inputs.
+- The methods `refreshTokenGrant`,`userTokenGrant` and `clientCredentialsGrant` of the `xsuaa-service` now accept a `string` or `XsuaaServiceCredentials` as first argument.
+A `string` value denotes the URL of the token service where it is not changed.
+A `XsuaaServiceCredentials` denotes that the XSUAA service is used to receive the token and a suffix `oauth/token` is appended to the URL property within `XsuaaServiceCredentials`.
```

Comment:
```suggestion
In case it is a `string` it is treated as the URL to fetch the access token from.
In case it is an object of type `XsuaaServiceCredentials`, the the URL is built by appending `oauth/token` to the `.url` of the given object.
```